mypy/nodes.py

```diff
@@ -742,7 +742,7 @@ def __init__(self, name: str, type: 'Optional[mypy.types.Type]' = None) -> None:
         super().__init__()
         self._name = name   # Name without module prefix
         # TODO: Should be Optional[str]
-        self._fullname = cast(Bogus[str], None)  # Name with module prefix
+        self._fullname = cast('Bogus[str]', None)  # Name with module prefix
```

Comment:
Why quotes are now needed here?

Reply:
Again, it's a micro-optimization. Profiling hints at spending a lot of time generating temporary type objects.